files/build_templates/sonic_debian_extension.j2

```diff
@@ -277,6 +277,9 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-ztp_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 {% endif %}
 
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/libdashapi_*.deb || \
```

Comment:
It will install extra packages for platforms not caring dash api. Is it possible to conditionally install?

Reply:
The question is still applicable.

Reply:
This would cause issues in the debug dump utility update which is done in sonic-utilities(https://github.com/sonic-net/sonic-utilities/pull/3387) As we perform imports from this debian file, and since the CLI for debug dump is not platform specific. Will check and see if it is possible to install based on platforms where we will use DASH objects.

Reply:
1. swss docker image install libdashapi, does not necessarily mean the host image need to install libdashapi on all platforms
2. the specific tool in sonic-utilities could be improved to conditional import this library

Reply:
Updated code to be platform specific for `nvidia-bluefield`